src/main/java/com/cleanroommc/groovyscript/api/documentation/annotations/RecipeBuilderDescription.java

```diff
@@ -51,4 +51,11 @@
      */
     int priority() default 1000;
 
+    /**
+     * The properties that should not be documented in this recipe builder, even though they exist in the class.
+     *
+     * @return the array of skipped properties
+     */
+    String[] skippedProperties() default {};
```

Comment:
this already exists via `@Property(property = "thing", needsOverride = true)`

Reply:
That's different. This is specifically attached to the Recipe Builder instead of its class to handle the case with having the same class. Although, thinking about it, the cleanest solution might be to have WorktableRecipeBuilderBase that gets inherited twice to obtain Shaped and Shapeless versions? That might be better than constantly checking for if(shaped). Let me know if I should change to that.

Reply:
you can put property annotations inside the builder documentation annotation